docs/installingLocal.md

```diff
@@ -83,3 +83,21 @@ cd /PrairieLearn
 ./docker/test_js.sh
 ./docker/test_python.sh
 ```
+
+### Connecting to an Existing Docker
```

Comment:
```suggestion
### Connecting to an existing Docker container
```

We've aren't totally consistent, but we are trying to standardize on sentence case for titles (leading caps only).